larpandoracontent/LArThreeDReco/LArTwoViewMatching/TwoViewTransverseTracksAlgorithm.cc

```diff
@@ -74,7 +179,8 @@ StatusCode TwoViewTransverseTracksAlgorithm::ReadSettings(const TiXmlHandle xmlH
     PANDORA_RETURN_RESULT_IF(STATUS_CODE_SUCCESS, !=, XmlHelper::ProcessAlgorithmToolList(*this, xmlHandle,
         "TrackTools", algorithmToolVector));
 
-    for (AlgorithmToolVector::const_iterator iter = algorithmToolVector.begin(), iterEnd = algorithmToolVector.end(); iter != iterEnd; ++iter)
+    for (AlgorithmToolVector::const_iterator iter = algorithmToolVector.begin(), iterEnd = algorithmToolVector.end(); 
```

Comment:
I swapped it back.
I see your point about auto, but it isn't really used elsewhere in the file, even in places where it could be used.  For consistency, I'm going to keep the strong typing.

Reply:
Sure, no issue with strong typing versus auto, I just figured the alternative loop construct itself was a bit more concise.